index.html

```diff
@@ -1,77 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- displays site properly based on user's device -->
 
   <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
-  
+  <link rel="stylesheet" href="style.css">
+
   <title>Frontend Mentor | Manage landing page</title>
 
   <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
   <style>
-    .attribution { font-size: 11px; text-align: center; }
-    .attribution a { color: hsl(228, 45%, 44%); }
+    .attribution {
+      font-size: 11px;
+      text-align: center;
+    }
+
+    .attribution a {
+      color: hsl(228, 45%, 44%);
+    }
   </style>
 </head>
-<body>
-
-  Pricing
-  Product
-  About Us
-  Careers
-  Community
 
-  Get Started
+<body>
+  <div id="header">
+    
+    <nav>
+      <ul>
+        
+
+       <li>
+        <img src="/images/logo.svg" />
+      </li> 
+        
+
+        <li>Pricing </li>
```

Comment:
The `<li>` should nest `<a>` like this:
`<li>``<a>`Pricing`</a>``</li>`

You get?